pkg/scheduler/manager.go

```diff
@@ -168,13 +168,32 @@ func (mgr *PodGroupManager) calculateAssignedPods(target *corev1.Pod) int {
 	return count
 }
 
-func (mgr *PodGroupManager) getCreationTimestamp(pod *corev1.Pod, defaultTime time.Time) time.Time {
+// rescheduleAfterAll is a method to reschedule pod group to end of queue.
+// the duration is the delay of current last pog group.
+func (mgr *PodGroupManager) rescheduleAfterAll(podGroup *apiv1.PodGroup, dur time.Duration) (*apiv1.PodGroup, error) {
+	podGroups, err := mgr.schedulingClient.PodGroups(podGroup.Namespace).List(mgr.ctx, metav1.ListOptions{})
```

Comment:
Here only get podGroups with the same namespace, it's not enough IMO.

Reply:
Yeah, I will fix it.

Reply:
Fixed